src/core/hle/service/es/es.cpp

```diff
@@ -55,7 +55,46 @@ class ETicket final : public ServiceFramework<ETicket> {
             {36, nullptr, "DeleteAllInactiveELicenseRequiredPersonalizedTicket"},
             {37, nullptr, "OwnTicket2"},
             {38, nullptr, "OwnTicket3"},
+            {501, nullptr, "Unknown"},
+            {502, nullptr, "Unknown2"},
             {503, nullptr, "GetTitleKey"},
+            {504, nullptr, "Unknown3"},
+            {508, nullptr, "Unknown4"},
```

Comment:
```suggestion
            {501, nullptr, "Unknown501"},
            {502, nullptr, "Unknown502"},
            {503, nullptr, "GetTitleKey"},
            {504, nullptr, "Unknown504"},
            {508, nullptr, "Unknown508"},
```
Etc for the entire function table